docker/sonarqube-aws-env/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+chown -R sonarqube:sonarqube $SONARQUBE_HOME
+
+if [ -z "$AWS_REGION" ]; then
```

Comment:
Some concern about this listed here: https://github.com/telia-oss/terraform-aws-grafana/pull/8#discussion_r209541364